erpnext/selling/doctype/sales_order/sales_order.json

```diff
@@ -3904,6 +3904,38 @@
    "set_only_once": 0, 
    "translatable": 0, 
    "unique": 0
+  }, 
+  {
```

Comment:
@PawanMeh these will be custom fields that will be created dynamically, not hard coded in the DocType

Reply:
@rmehta changes made as requested, also have pushed some more fixes in the same commit to fix issues as part of testing with larger set of data.